frontend/src/store/player-element.ts

```diff
@@ -355,12 +314,21 @@ class PlayerElementStore extends CommonStore<PlayerElementState> {
       }
     );
 
+    /**
+     * TODO: This should be where the subtitle track is changed, 
+     * and whether or not it is rendered on client or transcoded/burned in on the server.
+     */
+    watch(() => this.usePreciseSubtitles, async (newValue) => {
+      if (!videoContainerRef.value) return
+
+
+    });
+
     /**
      * We need to destroy JASSUB so the canvas can be recreated in the other view
      */
     watch(videoContainerRef, () => {
       if (!videoContainerRef.value) {
-        this._freeSsaTrack();
```

Comment:
This function has to be refactored for ass.js. It prevents multiple canvases from rendering at the same time when switching through tracks.

Reply:
Hey, believe this is working in latest commit.